### PR TITLE
Machine learning v1

### PR DESCRIPTION
**Pull request by serqiiiio**
Add Machine Learning Model

### CODE REVIEW

**Review by jchoque87 — CHANGES_REQUESTED** [review]
Add .gitignore file for node_modules

**Comment by serqiiiio** [comment]
@jchoque87 I alrady add the git ignore

**Review by jchoque87 — CHANGES_REQUESTED** [review]
add break line to .gitignore file

File: machineLearning/src/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+                objectSeacrched,
+                "al",
+                percentage * 100,
+                "%"
+            );
+            const dataSelected = {
+                model: "cocoSsd",
+                objectS: objectSeacrched,
+                percetn: percentage,
+                quantity: number // file name and time
```
Comment (MrRoaming) [inline machineLearning/src/index.js:45]: I have a doubt in here, can you explain to me what does 'file name and time' means? Please

Comment (serqiiiio) [reply machineLearning/src/index.js:45]: This is the JSON object that the Machine Learning End point needs this object to send it

Comment (MrRoaming) [reply machineLearning/src/index.js:45]: Thanks Sergio, but I was talking about the comment. **_"quantity: number // file name and time"_** does the _**quantity**_ has those values on it?

**Pull request merged**